registrations/models.py

```diff
@@ -265,14 +265,12 @@ def can_be_edited_by(self, user):
         """Check if current registration can be edited by the given user"""
         return (
             user.is_superuser
-            or user.is_admin_of(self.event.publisher)
-            or user.is_registration_admin_of(self.event.publisher)
+            or user.is_admin_of(self.publisher)
+            or user.is_registration_admin_of(self.publisher)
         )
 
     def is_user_editable_resources(self):
```

Comment:
This `Registration` method does not seem to be used anywhere - can be removed most probably.

Reply:
Removed the unused method.